types/siren-parser/superagent.d.ts

```diff
@@ -2,5 +2,5 @@ import { Response, SuperAgentRequest } from "superagent";
 import { Action, Entity } from "./index";
 
 export function parse(res: string): Entity;
-export function parse(res: Response, fn: (err: Error | null, body: Entity) => void): void;
+export function parse(res: Response, fn: (err: Error | null, body: any | Entity) => void): void;
```

Comment:
Union with `any` is like multiplying by 0.
```suggestion
export function parse(res: Response, fn: (err: Error | null, body: any) => void): void;
```